### PR TITLE
Updated `largo_home_single_top` function

### PR DESCRIPTION
**Pull request by joshdarby**
Now gets `homepage_feature_term` and `top_story_term` values from slug
instead of by name for issue #1445.

### CODE REVIEW

File: homepages/homepage.php [hunk 1]
```diff
@@ -94,8 +94,8 @@ function largo_home_single_top() {
 	$big_story = null;
 
 	// Cache the terms
-	$homepage_feature_term = get_term_by( 'name', __('Homepage Featured', 'largo'), 'prominence' );
-	$top_story_term = get_term_by( 'name', __('Homepage Top Story', 'largo'), 'prominence' );
+	$homepage_feature_term = get_term_by( 'slug', __('homepage-featured', 'largo'), 'prominence' );
+	$top_story_term = get_term_by( 'slug', __('top-story', 'largo'), 'prominence' );
```
Comment (benlk) [inline homepages/homepage.php]: Are the slugs translated elsewhere in Largo?

Comment (joshdarby) [reply homepages/homepage.php]: @benlk When I was working on this I did a full search for `get_term_by( 'name', __('Homepage Top Story', 'largo'), 'prominence' );` and `get_term_by( 'name', __('Homepage Featured', 'largo'), 'prominence' );` and the only results were in this file.

Comment (benlk) [reply homepages/homepage.php]: Not quit what I was asking. In places like https://github.com/INN/largo/blob/72cc8433a1be1de4f22789e1ec89496d54a653d9/homepages/layouts/HomepageSingleWithFeatured.php#L15-L21 we use the term slug as a bare string, but this PR runs the slug through the translation function to localize the term's slug. Do we do that anywhere else in Largo?

Comment (joshdarby) [reply homepages/homepage.php]: @benlk Ah, sorry. I don't see the slug being localized anywhere else. Good catch. I'll remove it and just use it as a regular string.

**Review by benlk — APPROVED** [review]

**Pull request merged**